apps/nextjs-website/src/config.ts

```diff
@@ -23,6 +28,16 @@ export const amplifyConfig = {
   authenticationFlowType: 'CUSTOM_AUTH',
 };
 
+// a BuildEnv instance ready to be used
+export const buildEnv = pipe(
+  makeBuildConfig(process.env),
+  E.map(makeBuildEnv),
+  E.getOrElseW((errors) => {
+    // eslint-disable-next-line functional/no-throw-statements
+    throw errors;
+  })
+);
+
```

Comment:
This file is going to be deprecated (as the comment on top of the file says), so you should not add stuff here.

Moreover, the `buildEnv` is an environment used during build time, so I think that if you use it on client side, this doesn't work as expected
```suggestion
```